### PR TITLE
Feature/chain of responsibility

### PR DESCRIPTION
**Pull request by lukasziukas**

### CODE REVIEW

File: Client/NoOpRunner.Client/MouseClickHandlers/PlayerHandler.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    
+                mediaPlayer.Open(ResourcesUriHandler.GetCharacterClickSound());
+
+                mediaPlayer.MediaEnded += (o, a) =>
+                {
+                    mediaPlayer.Position = TimeSpan.Zero;
+                    mediaPlayer.Stop();
+                };
+                
+                mediaPlayer.Volume = 0.5;
```
Comment (Kodnot) [inline Client/NoOpRunner.Client/MouseClickHandlers/PlayerHandler.cs]: Currently the volume level is not tied to our volume settings, right? How problematic would it be to do so?

File: Core/NoOpRunner.Core/Controls/InputHandlerImplementorPlayerTwo.cs.orig [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using NoOpRunner.Core.Enums;
+
+namespace NoOpRunner.Core.Controls
+{
+    public class InputHandlerImplementorPlayerTwo : InputHandlerImplementor
+    {
+        private PlayerTwo playerTwo;
+        
+        public InputHandlerImplementorPlayerTwo(PlayerTwo playerTwo)
```
Comment (Kodnot) [inline Core/NoOpRunner.Core/Controls/InputHandlerImplementorPlayerTwo.cs.orig]: ? Accidental merge artifacts?

**Review by Kodnot — CHANGES_REQUESTED** [review]

**Comment by Kodnot** [comment]
After you spawn some saws, some new generated powerups may use the saw sprite for some reason

File: Core/NoOpRunner.Core/Controls/InputHandlerImplementorPlayerTwo.cs.orig (hunk 2 above)
Comment (lukasziukas) [reply Core/NoOpRunner.Core/Controls/InputHandlerImplementorPlayerTwo.cs.orig]: Deleted

**Comment by lukasziukas** [comment]
sinkfunny

**Comment by lukasziukas** [comment]
Sprite fixed

File: Core/NoOpRunner.Core/NoOpRunner.Core.csproj [hunk 3]
```diff
@@ -47,23 +47,24 @@
...
     <Compile Include="Configurators\ShapeConfigurator.cs" />
-    <Compile Include="Constants\MediatorEvents.cs" />
+    <!-- <Compile Include="Constants\MediatorEvents.cs" /> -->
     <Compile Include="Controls\InputHandlerImplementor.cs" />
     <Compile Include="Controls\InputHandlerImplementorPlayerOne.cs" />
     <Compile Include="Controls\InputHandlerImplementorPlayerTwo.cs" />
     <Compile Include="DisposedObjectsPool.cs" />
     <Compile Include="Dtos\CoordinatesDto.cs" />
-    <Compile Include="Dtos\MediatorMessageDto.cs" />
+    <!-- <Compile Include="Dtos\MediatorMessageDto.cs" /> -->
```
Comment (Kodnot) [inline Core/NoOpRunner.Core/NoOpRunner.Core.csproj:63]: These can be removed

**Review by Kodnot — APPROVED** [review]

**Comment by lukasziukas** [comment]
Sprites fixed, sound don't care

**Pull request merged**